activiti-cloud-notifications-graphql-service/services/events/src/main/java/org/activiti/cloud/services/notifications/graphql/events/consumer/EngineEventsConsumerMessageHandler.java

```diff
@@ -44,25 +44,25 @@ public EngineEventsConsumerMessageHandler(Transformer transformer,
         this.transformer = transformer;
     }
 
-    @StreamListener
-    public void receive(@Input(EngineEventsConsumerChannels.SOURCE)
-                            Flux<Message<List<Map<String,Object>>>> input) {
+    @StreamListener(EngineEventsConsumerChannels.SOURCE)
```

Comment:
If I leave the @Input annotation I get this exception: A method annotated with @StreamListener may use @Input or @Output annotations only in declarative mode and for parameters that are binding targets or convertible from binding targets

Reply:
@jesty @erdemedeiros Unfortunately, this changes behavior in a major way.  The intention was to use Flux as pass-through from underlying message channel that automatically handles reconnects due to broker failures. I will try to find another approach to do that....